v4/core/request_builder.go

```diff
@@ -47,8 +47,9 @@ const (
 	CONTENT_TYPE            = "Content-Type"
 	FORM_URL_ENCODED_HEADER = "application/x-www-form-urlencoded"
 
-	ERRORMSG_SERVICE_URL_MISSING = "The service URL is required."
-	ERRORMSG_SERVICE_URL_INVALID = "There was an error parsing the service URL: %s"
+	ERRORMSG_SERVICE_URL_MISSING = "service URL is empty"
```

Comment:
I changed these error message text strings to sort of get them more aligned with the Go style (incomplete sentences, etc.).